zettarepl/transport/zfscli/exception.py

```diff
@@ -116,3 +116,9 @@ def __exit__(self, exc_type, exc_val, exc_tb):
             )
         ):
             raise RecoverableReplicationError(str(exc_val)) from None
+
+        if (
+            isinstance(exc_val, ExecException) and
+            re.search(r"cannot send .+: snapshot data/src@2018-10-01_01-00 does not exist", exc_val.stdout)
```

Comment:
I dont think you mean this exact regex, with the dataset name and snapshot name baked in?